vesta/data/lang.yml

```diff
@@ -1,35 +1,39 @@
 fr:
+  general_yes: "🟩 Oui"
+  general_no: "🟥 Non"
   invalid_link: Le lien n'est pas valide
   invalid_image_link: Le lien de votre image n'est pas valide
   command_created: La commande a bien été créée !
-  custom_form: Création d'une commande custom
+  custom_form: Création d'une commande personnalisée
   custom_form_keyword: Mot clé de la commande
   custom_form_title: Titre du message
   custom_form_content: Contenu du message
   custom_form_link: Lien
   custom_form_image: Image
   custom_form_color: Couleur de l'embed
-  command_already_exist: La commande existe déjà
+  command_already_exist: Cette commande existe déjà
   command_not_exist: Cette commande n'existe pas
-  command_deleted: La custom command à bien été supprimée
+  command_deleted: La commande personnalisée a bien été supprimée
   list_commands: Liste des commandes
   list_commands2: Partie 2
-  too_much_commands: Il y a déjà trop de custom commands
+  too_much_commands: Il y a déjà trop de commandes personnalisées
   unexpected_error: Une erreur s'est produite
-  permissions_error: Vous n'avez pas les permissions pour faire cette commande
-  config_review: Le salon de review à bien été config
-  config_projects: Le salon de projets à bien été config
-  config_lang: La langue à bien été changée
-  nickname_banned: Vous avez été banni du système de rename
-  nickname_incorrect_title: ⚠️Pseudo incorrect !
-  nickname_incorrect_description: "Ce nom n'est pas valide, merci d'entrer un nom validant la regex suivante : "
-  nickname_changed: Votre pseudo à été changé avec succès !
-  bot_permissions_error: "Malheureusement, le bot n'a pas les permissions suffisantes pour faire cela, merci de demander à vos administrateurs de lui fournir les suivantes :"
-  nickname_ban: a bien été banni de la commande nickname
-  nickname_not_banned: n'est pas banni de la commande nickname
-  nickname_unban: a bien été débanni de la commande nickname
-  nickname_list_title: Membres bannis du nickname
-  presentations_not_available: Le systeme de présentation n'a pas été activé sur votre serveur
+  permissions_error: Vous n'avez pas les permissions pour exécuter cette commande
+  config_review: Le salon de revue à bien été modifié
```

Comment:
No, please stop trying to push your lang commit at the same time as the coc module.
And we won't say "le salon de revue", just who is saying that? It's okay do do frenchglish sometimes

There are a lot of other problems, but I won't show them all for this file

Reply:
Wrong branch upsy... Will fix by moving the commit